docs/develop/01_sdk/02_cookbook/01_dids/05_full_did_query.md

```diff
@@ -0,0 +1,21 @@
+---
+id: did-query
+title: Resolve a DID
+---
+
+import TsJsBlock from '@site/src/components/TsJsBlock';
+
+import FullDidQuery from '!!raw-loader!@site/code_examples/core_features/did/06_full_did_query.ts';
+
+Querying the state of a DID is also called **resolution**.
+
+Resolving a DID to its DID Document can be done via the KILT resolver.
```

Comment:
What's a KILT resolver? Is that the SDK?

Reply:
Good point. Is this better? Or do you expect a more thorough explanation? https://github.com/KILTprotocol/docs/pull/174/commits/6afa0cf964c68da30bd360e5e9eb5a89be532e59